source/_changelogs/2017-12-01-January.md

```diff
@@ -0,0 +1,32 @@
+---
+title: January 2017
+changelog: true
+---
+## Platform Improvements
+###Terminus 1.0
+After months of retooling and feedback from users, we're excited to officially release Terminus 1.0! This release comes with a commitment to backwards compatibility for scripts and plugins, as well as a new core engine based on Symfony Console. Everything you've been doing with Terminus is included in this new version with more additions continually coming! [Learn more about this new feature](https://pantheon.io/features/command-line-interface) and check out the [Terminus Manual](https://pantheon.io/docs/terminus).
+
+###Terminus Plugins
+The open plugin architecture of Terminus allows developers to extend it’s functionality and [create new commands](https://pantheon.io/docs/terminus/plugins/create). For a list of supported plugins, take a look at our new [Plugin Directory](https://pantheon.io/docs/terminus/plugins/directory) in the docs.
+
+**Mass Updates**
+[The Mass Update plugin](https://github.com/pantheon-systems/terminus-mass-update) applies all available upstream updates to a list of sites. This allows users who manage a lot of sites to easily stay on top of Drupal and WordPress core updates.
+
+**Build Tools**
+[This Terminus plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin) contains a collection of six additional commands useful during the build step on a Pantheon site that uses a GitHub PR workflow.
+
+**Filer**
+[Terminus Filer](https://github.com/terminus-plugin-project/terminus-filer-plugin) is named as such because it’s an easy way to launch your SFTP client with all the connection details “filled” in.
+
+**Rsync**
+The [Terminus Rsync Plugin](https://github.com/pantheon-systems/terminus-rsync-plugin) makes it easy to copy files between your local computer and Pantheon. Once installed, you can use rsync as you normally would to transfer files, wrapped in Terminus commands.
+
+###WordPress upgrade to 4.7.2
```

Comment:
suggest putting version number and CMS name together:
e.g. WordPress 4.7.2 or Drupal 8.2.5 like in https://pantheon.io/docs/changelog/#december-2016